docs/user/accumulo/configuration.rst

```diff
@@ -40,6 +40,39 @@ The timeout is defined as a duration, e.g. ``60 seconds`` or ``100 millis``. See
 
 __ https://accumulo.apache.org/1.9/apidocs/org/apache/accumulo/core/client/BatchWriterConfig.html#setTimeout(long,%20java.util.concurrent.TimeUnit)
 
+Remote Processing Properties
+----------------------------
+
+The following properties control the push-down processing for certain queries to the Accumulo tablet servers, vs
```

Comment:
Hmm...  I wonder if this would read better as 

"The following properties control the push-down processing for certain queries to the Accumulo tablet servers versus processing in the client."

I'm trying to figure out if the comma here is good or bad...